kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -238,7 +239,9 @@ def _create_kubelet_prometheus_instance(self, instance):
         This is so the base class can create a scraper_config with the proper values.
         """
         kubelet_conn_info = get_connection_info()
-        endpoint = kubelet_conn_info.get('url')
+
+        endpoint = kubelet_conn_info.get('url') if kubelet_conn_info is not None else "dummy_url/kubelet" # dummy needed in case get_connection_info isn't running when the check is first accessed
+        endpoint = "dummy_url"
```

Comment:
Possible holdover from testing? Because endpoint will always be `dummy_url`, making the line above this unnecessary